### PR TITLE
At the helm with Ships Mahindra

### PR DESCRIPTION
**Pull request by TeganAli**
# About

This is a post about...

# Release

Is this post for a specific release? If so which one?

# Blog image idea

The design team will work on a design for the blog image. Add any ideas you have about the graphic image here.

# Pre-requisites

- [ ] The draft is complete and this post is ready to be reviewed.
- [ ] I'm familiar with the [writing for Octopus TL;DR](https://www.octopus.design/writing/writing-tldr/).
- [ ] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [ ] The images I've included follow the [rules for working with images](https://www.octopus.design/writing/working-with-images/).
- [ ] I've added alt text for all the images I've included (alt text describes image to people unable to see it, 125 characters max)
- [ ] One screenshot (if any are included) is suitable for Twitter. This screenshot should be 16:9 ratio and at least 1200 x 675 pixels. For screenshots from the Octopus Web Portal, please use Chrome and the browser’s zoom function to remove as much white space as possible. If screenshots are annotated (e.g., with boxes or arrows), please also provide a version that is unannotated for use on Twitter. Do not use dark mode for the Octopus Web Portal.

### CODE REVIEW

File: blog/2022-06/at-the-helm-ships-mahindra/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## How do you decide what issues and solutions to tackle next?
+
+Ah… the art and science of prioritization. There must be millions of articles written on this subject. Every PM develops their own process and chooses their own set of tools and frameworks. 
+
+I like to gather inputs from various sources and analyze them - customer feedback, usage data, market trends. I listen for cues and patterns, then frame these as opportunities to pursue. Then I reflect on our goals and the next best bet to get us closer to one of our goals. 
+
+Our long term objective for Config as Code is to reach at least 60% feature adoption. So the key question is - what's stopping users from adopting Config as Code today? After we have a list of opportunities, I weigh up which will add enough value to encourage our users to adopt Config as Code. Do we have enough data or signals from our users and the market to make a bet on these opportunities? If we do, we start working on that problem. If not, we find small experiments we can run on our assumptions. This includes more user research, testing designs, testing prototypes, or shipping minimal features.
```
Comment (TeganAli) [inline blog/2022-06/at-the-helm-ships-mahindra/index.md:42]:
```suggestion
Our long term objective for Config as Code is to reach at least 60% feature adoption. So the key question is - what's stopping users from adopting Config as Code today? After we have a list of opportunities, I weigh up which will add enough value to encourage our users to adopt Config as Code. Do we have enough data or signals from our users and the market to make a bet on those opportunities? If we do, we start working on that problem. If not, we find small experiments we can run on our assumptions. This includes more user research, testing designs, testing prototypes, or shipping minimal features.
```

**Comment by shipra-octopus** [comment]
Looks great Tegan :-) Thank you!
Just a small feedback, I would change this -
> If not, we find small experiments we can run on our assumptions
to this - 
> If not, we find small experiments we can run _to test_ our assumptions

**Comment by TeganAli** [comment]
I'll fix that up now, thanks so much, @shipra-octopus :) 
Then are you happy for me to publish it any time? (I'll publish it well ahead of the publication date so I can test the link in the newsletter.)

File: blog/2022-06/at-the-helm-ships-mahindra/index.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## How do you decide what issues and solutions to tackle next?
+
+Ah… the art and science of prioritization. There must be millions of articles written on this subject. Every PM develops their own process and chooses their own set of tools and frameworks. 
+
+I like to gather inputs from various sources and analyze them - customer feedback, usage data, market trends. I listen for cues and patterns, then frame these as opportunities to pursue. Then I reflect on our goals and the next best bet to get us closer to one of our goals. 
+
+Our long term objective for Config as Code is to reach at least 60% feature adoption. So the key question is - what's stopping users from adopting Config as Code today? After we have a list of opportunities, I weigh up which will add enough value to encourage our users to adopt Config as Code. Do we have enough data or signals from our users and the market to make a bet on those opportunities? If we do, we start working on that problem. If not, we find small experiments we can run on our assumptions. This includes more user research, testing designs, testing prototypes, or shipping minimal features.
```
Comment (TeganAli) [inline blog/2022-06/at-the-helm-ships-mahindra/index.md:42]:
```suggestion
Our long term objective for Config as Code is to reach at least 60% feature adoption. So the key question is - what's stopping users from adopting Config as Code today? After we have a list of opportunities, I weigh up which will add enough value to encourage our users to adopt Config as Code. Do we have enough data or signals from our users and the market to make a bet on those opportunities? If we do, we start working on that problem. If not, we find small experiments we can run to test our assumptions. This includes more user research, testing designs, testing prototypes, or shipping minimal features.
```

File: blog/2022-06/at-the-helm-shipra-mahindra/index.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+---
+title: At the helm with Shipra Mahindra
+description: A series where we chat to people at Octopus about their role, what they’re working on to improve the product, and more. Hear from Shipra Mahindra, Senior Product Manager.
+author: ships.mahindra@octopus.com
+visibility: public
+published: 2022-06-21-1400
```
Comment (TeganAli) [inline blog/2022-06/at-the-helm-shipra-mahindra/index.md:6]:
```suggestion
published: 2022-06-20-1400
```

**Pull request merged**